cypress-tests/cypress/e2e/updateProjects.cy.ts

```diff
@@ -0,0 +1,109 @@
+import { v4 as uuidv4 } from "uuid";
+import { TIMEOUTS } from "../../config";
+
+
+const username = Cypress.env("TEST_USERNAME");
+
+const projects = {
+  shouldFork: true,
+  namespace: "renku-ui-tests",
+  v7: "renku-project-v7",
+  v8: "renku-project-v8",
+}
```

Comment:
I think a comment after this explaining how to modify the configuration to aid local debugging would be good.

Reply:
Goog point! I added a comment